packages/non-core/coffeescript/package.js

```diff
@@ -1,14 +1,14 @@
 Package.describe({
   summary: "Javascript dialect with fewer braces and semicolons",
-  version: "1.12.3_1"
+  version: "1.12.6_1"
```

Comment:
Shall we remove the `_1` in the version number, since we're releasing for 1.12.6 for the first time?